writers/streamingbatchwriter/streamingbatchwriter_test.go

```diff
@@ -197,7 +197,7 @@ func TestStreamingBatchSizeRows(t *testing.T) {
 	}()
 
 	table := schema.Table{Name: "table1", Columns: []schema.Column{{Name: "id", Type: arrow.PrimitiveTypes.Int64}}}
-	record := array.NewRecord(table.ToArrowSchema(), nil, 0)
+	record := getRecord(table.ToArrowSchema(), 1)
```

Comment:
What's the reason we need to change this test? Do we not support records without rows now?

Reply:
Ah maybe because we were always incrementing the row count regardless if the record has rows or not

Reply:
yeah, we had records with 0 rows in tests